### PR TITLE
fix(paloalto/ngfw): Improve smart descriptions and add relationships.

### PR DESCRIPTION
**Pull request by PierrickV**

### CODE REVIEW

File: Palo Alto Networks/paloalto-ngfw/_meta/smart-descriptions.json [hunk 1]
```diff
@@ -29,26 +29,27 @@
...
-                "field": "paloalto.PanOSQuarantineReason"
+                "source": "destination.ip",
+                "target": "host.name",
+                "type": "associated with"
             },
             {
-                "field": "event.reason"
+                "source": "process.name",
+                "target": "running on",
+                "type": "source.ip"
```
Comment (kisscool-fr) [inline Palo Alto Networks/paloalto-ngfw/_meta/smart-descriptions.json:52]: Just a question: why `target` and `type` are invert (regarding the other field above) ? 🤔

**Review by kisscool-fr — COMMENTED** [review]
Just a simple question ⬇️

File: Palo Alto Networks/paloalto-ngfw/_meta/smart-descriptions.json (hunk 1 above)
Comment (PierrickV) [reply Palo Alto Networks/paloalto-ngfw/_meta/smart-descriptions.json]: Nice catch ! It should be fixed in latest commit

**Review by kisscool-fr — APPROVED** [review]
Sounds better 👏🏼

**Review by yosradirgham — APPROVED** [review]

**Pull request merged**